raiden/settings.py

```diff
@@ -70,6 +70,11 @@
 DEFAULT_WAIT_BEFORE_LOCK_REMOVAL = BlockTimeout(2 * DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS)
 DEFAULT_CHANNEL_SYNC_TIMEOUT = 5
 
+DEFAULT_AVERAGE_BLOCK_TIME = 7.0
```

Comment:
I assume this intentionally underestimates the blocktime to be on the safe side? 
If so then a comment would probably make sense.

OTOH this is also highly dependent on the chain used...

Reply:
Yes, I can also set it to `15`s and take the extra time in the formula for the timeout